docs/troubleshooting.md

```diff
@@ -1,84 +1,294 @@
 ```
-sudo port install openssl
+
+in this case, use the file menu in VSCode and choose `Help`>`About`
+(or `Code`>`About Visual Studio Code` on macOS) to get version information.
+
+### PowerShell Extension Version
+
+[Your installed PowerShell Extension version] can similarly be found with:
+
+```shell
+code --list-extensions --show-versions
 ```
 
-You will need to take an additional step once installation completes:
+With VSCode Insiders:
 
+```shell
+code-insiders --list-extensions --show-versions
 ```
-sudo ln -s /opt/local/lib/libcrypto.1.0.0.dylib /usr/local/lib/libcrypto.1.0.0.dylib
-sudo ln -s /opt/local/lib/libssl.1.0.0.dylib /usr/local/lib/libssl.1.0.0.dylib
+
+You should get an output like:
+
+```text
+DavidAnson.vscode-markdownlint@0.20.0
+eamodio.gitlens@8.5.6
+EditorConfig.EditorConfig@0.12.4
+jchannon.csharpextensions@1.3.0
+k--kato.docomment@0.1.2
+ms-vscode.cpptools@0.18.1
+ms-vscode.csharp@1.16.1
+ms-vscode.PowerShell@2.0.0
+twxs.cmake@0.0.17
+vscodevim.vim@0.16.5
 ```
 
-Thanks to [@MarlonRodriguez](https://github.com/MarlonRodriguez) for the tip!
+If VSCode isn't on your path use the [Command Palette]
+(<kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>P</kbd>) to enter
+`Extensions: Show Installed Extensions` and list your extensions.
+
+### Editor Services Version
+To get the [PowerShell Editor Services] version, in the Integrated
+Console, enter:
+
+```powershell
+> $psEditor.EditorServicesVersion
+Major  Minor  Build  Revision
+-----  -----  -----  --------
+1      8      4      0
+```
+
+### PowerShell Version Table
+You can get [your PowerShell version table] from the Integrated Console:
+
+```powershell
+> $PSVersionTable
+Name                           Value
+----                           -----
+PSVersion                      6.1.0
+PSEdition                      Core
+GitCommitId                    6.1.0
+OS                             Microsoft Windows 10.0.18242
+Platform                       Win32NT
+PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
+PSRemotingProtocolVersion      2.3
+SerializationVersion           1.1.0.1
+WSManStackVersion              3.0
+```
+
+### Operating System Information
+
+- Windows - all needed information should already be in the `$PSVersionTable`
+- macOS
+  - Your macOS version (e.g. High Sierra 10.13.6)
+- Linux
+  - `uname -a`
+  - Your distro and version (usually `lsb_release -a` is the best here)
+
+### Note on Security
+
+If you believe there is a security vulnerability in the [PowerShell Extension]
+(or in [PowerShell Editor Services]), it **must** be reported directly to
+secure@microsoft.com to allow for [Coordinated Vulnerability Disclosure].
+**Only** open an issue if secure@microsoft.com has confirmed that filing
+an issue on GitHub is appropriate.
+
+[Editor Syntax]: https://github.com/PowerShell/EditorSyntax
+[PowerShell Editor Services]: https://github.com/PowerShell/PowerShellEditorServices
+[PowerShell Extension]: https:github.com/PowerShell/vscode-powershell/
```

Comment:
This link doesn't resolve, needs`https:githib.com` > `https://github.com`

Side note, I didn't know about this syntax! Really cool 😃

Reply:
Aha! I wondered why it was breaking